uaclient/messages/__init__.py

```diff
@@ -1028,15 +1028,41 @@ class TxtColor:
 
 FIPS_UPDATES_TITLE = "FIPS Updates"
 FIPS_UPDATES_DESCRIPTION = (
-    "NIST-certified core packages with priority security updates"
+    "NIST-certified FIPS crypto packages with priority security updates"
 )
 FIPS_UPDATES_HELP_TEXT = """\
-fips-updates installs fips modules including all security patches for those
-modules that have been provided since their certification date. You can find
-out more at {url}""".format(
+fips-updates installs FIPS 140 crypto packages including all security patches
+for those modules that have been provided since their certification date.
+You can find out more at {url}""".format(
     url=urls.FIPS_HOME_PAGE
 )
 
+FIPS_PREVIEW_TITLE = "FIPS Preview"
+FIPS_PREVIEW_DESCRIPTION = "crypto package under certification with NIST"
+FIPS_PREVIEW_HELP_TEXT = """\
+Installs FIPS crypto packages that are under certification with NIST,
+for FedRAMP, FISMA and compliance use cases."""
+PROMPT_FIPS_PREVIEW_PRE_ENABLE = (
+    """\
+This will install crypto packages that have been submitted to NIST for review
+but do not have FIPS certification yet. Use this for early access to the FIPS
+modules.
+Please note that the Livepatch service will be unavailable after
+this operation.
+Warning: This action can take some time and cannot be undone.
+"""
+    + PROMPT_YES_NO
+)
+PROMPT_FIPS_PREVIEW_PRE_ENABLE = (
+    """\
+This will install not NIST-certified FIPS packages.
+Please use this service only for test purposes.
+Additionally, the Livepatch service will be unavailable after the operation.
+Warning: This action can take some time and cannot be undone.
+"""
+    + PROMPT_YES_NO
+)
+
```

Comment:
I think this is the old version and is overwriting the new one above. The new one above this one looks nicer.